src/celutil/utf8.h

```diff
@@ -11,7 +11,8 @@
 #define _CELUTIL_UTF8_
 
 #include <string>
-#include <wchar.h>
+#include <vector>
+#include <climits>
```

Comment:
You have moved wchar.h instead of climits.h

Reply:
Oh... I wasn't sure, which one should be moved.